dotCMS/src/main/webapp/html/portlet/ext/sitesearch/site_search_audit.jsp

```diff
@@ -57,11 +56,19 @@
     </tr>
     <% for(SiteSearchAudit a : recents) { %>
     <%
-        StringBuilder hostList=new StringBuilder();
-        for(String hid : a.getHostList().split(",")) {
-            if(UtilMethods.isSet(hid))
-                hostList.append(APILocator.getHostAPI().find(hid, APILocator.getUserAPI().getSystemUser(), false).getHostname())
-                        .append("  ");
+        final StringBuilder hostList = new StringBuilder();
+        final String [] sites = a.getHostList().split(",");
+        if(UtilMethods.isSet(sites) ){
```

Comment:
sites must check if size is >= 1

Reply:
UtilMethods.isSet(String) does that internally